package.json

```diff
@@ -96,8 +96,9 @@
     }
   },
   "lint-staged": {
-    "**/src/**/*.{tsx,jsx,ts,js}": [
-      "yarn lint:write"
+    "**/*.{mdx,js,jsx,ts,tsx}": [
+      "eslint --fix --max-warnings 0",
+      "prettier --write"
```

Comment:
`lint:write` should be running eslint & prettier. could you pls explain the reasoning behind doing it like this instead?

https://github.com/alchemyplatform/aa-sdk/blob/eae95ed42b1d405133d1363c33dd28d587af923e/package.json#L49